rules/react.js

```diff
@@ -104,5 +104,195 @@ module.exports = {
     // Some legacy code heavy use indexes for keys. It is complicated to convert to something else
     // expecially when data is client side derived.
     'react/no-array-index-key': 'warn',
+
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md
+    // AirBnB
+    // 'react/jsx-max-props-per-line': [
+    //   'error',
+    //   {
+    //     maximum: 1,
+    //     when: 'multiline',
+    //   },
+    // ],
+    //
+    // Set to "always" so that new props will always on new line if there are more than 1 prop
+    'react/jsx-max-props-per-line': [
```

Comment:
Did you manage to fix this so it works with prettier?

Reply:
Oops.. missed that documentation in eslint-config-prettier.
Moved to "TODO" section to be enabled when we moved into prettier-eslint to do the formatting.

Reply:
@tavurth